db/controllers/api.controller.js

```diff
@@ -0,0 +1,7 @@
+const endpoints = require("../../endpoints.json")
+const fs = require('fs')
+
+exports.getJson = (req, res, next) => {
+        res.status(200)
+        .send(JSON.parse(fs.readFileSync('endpoints.json', "utf-8")))
```

Comment:
Couple of things using readFileSync could be a problem as it's synchronous code that could become blocking if the file was very large